### PR TITLE
Updated a few things

### PR DESCRIPTION
**Pull request by Fortifier42**
Expanded MCMMO party

### CODE REVIEW

File: src/main/java/net/gnomeffinway/depenizen/support/plugins/McMMOSupport.java [hunk 1]
```diff
@@ -1,56 +1,16 @@
...
-    @Override
-    public String additionalTags(Attribute attribute) {
-
-        if (attribute.startsWith("party")) {
-
-            if (!attribute.hasContext(1) || PartyManager.getParty(attribute.getContext(1)) == null)
-                return null;
-
-            Party party = PartyManager.getParty(attribute.getContext(1));
-            attribute = attribute.fulfill(1);
```
Comment (mcmonkey4eva) [inline src/main/java/net/gnomeffinway/depenizen/support/plugins/McMMOSupport.java]: this section is useful to keep

File: pom.xml [hunk 2]
```diff
@@ -156,13 +157,13 @@
...
-			<scope>system</scope>
-			<systemPath>${basedir}/src/main/resources/Heroes.jar</systemPath>
-		</dependency>
+        <dependency>
+            <groupId>com.herocraftonline</groupId>
+            <artifactId>Heroes</artifactId>
+            <version>1.5.57</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/src/main/resources/Heroes.jar</systemPath>
+        </dependency>
```
Comment (mcmonkey4eva) [inline pom.xml]: "Lemme just randomize all the spacing here..."

File: pom.xml [hunk 3]
```diff
@@ -1,11 +1,12 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <groupId>net.gnomeffinway.depenizen</groupId>
     <artifactId>Depenizen</artifactId>
     <version>0.2.2</version>
     <name>Depenizen</name>
     <url>https://github.com/Morphan1/Depenizen</url>
```
Comment (mcmonkey4eva) [inline pom.xml]: you should fix this if you're already editing the pom :O

File: pom.xml (hunk 2 above)
Comment (Fortifier42) [inline pom.xml]: It was half tabs, half spaces.

**Comment by Morphan1** [comment]
I guess I'll assume this all works

**Pull request merged**